src/insights/balance.ts

```diff
@@ -72,7 +72,7 @@ export interface QueryGetBalanceSupplyResponse {
   entries: TokenBalance[];
 }
 
-export interface QueryGetBalanceDistributionRequest extends QueryByTimeRequest {}
+export type QueryGetBalanceDistributionRequest = QueryByTimeRequest
```

Comment:
Hmm is it possible to enter an empty object? Maybe eslint-disable this line or find the property that does not allow empty objs.

Also, do that for all the other insights `QueryXX` structs

Reply:
Hmm, not sure if the eslint was updated since this comment but, 
![image](https://github.com/Switcheo/carbon-js-sdk/assets/47170358/f558d0dc-2b7d-4a9a-9e28-11e46b92a37a)
There no longer seems to be an es-lint error anymore when using empty structs.